components/crud-web-apps/tensorboards/Dockerfile

```diff
@@ -48,4 +48,4 @@ COPY ./tensorboards/backend/Makefile .
 
 COPY --from=frontend /src/dist/ /src/app/static/
 
-ENTRYPOINT make run
+ENTRYPOINT gunicorn -w 3 --bind 0.0.0.0:5000 --access-logfile - entrypoint:app

```

Comment:
Can we use the `/bin/bash -c` notation here so that
1. we explicitly use `bash` instead of `sh` (https://docs.docker.com/engine/reference/builder/#shell-form-entrypoint-example), and
2. it is uniform with what we do for other web apps (JWA https://github.com/kubeflow/kubeflow/blob/28af7716bc22f25158ddde05af58d53f4fafc939/components/crud-web-apps/jupyter/Dockerfile#L50)

Reply:
Makes sense, will add a commit for this

Reply:
pushed a commit, will resolve this